js/flightlog_parser.js

```diff
@@ -806,64 +806,111 @@ var FlightLogParser = function(logData) {
                     var expoParams = parseCommaSeparatedString(fieldValue);
                     that.sysConfig.superExpoFactor    = expoParams[0];
                     that.sysConfig.superExpoFactorYaw = expoParams[1];
-
                 } else {
                     that.sysConfig.superExpoFactor = parseInt(fieldValue, 10);
                 }
             break;
 
             /* CSV packed values */
+
             case "rates":
+                that.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue);
+            break;
             case "rate_limits":
+                that.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue);
+            break;
+
             case "rollPID":
+                that.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue);
+            break;
             case "pitchPID":
+                that.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue);
+            break;
             case "yawPID":
+                that.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue);
+            break;
+
             case "altPID":
+                that.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue);
+            break;
             case "posPID":
+                that.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue);
+            break;
             case "posrPID":
+                that.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue);
+            break;
             case "navrPID":
+                that.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue);
+            break;
+
             case "levelPID":
+                that.sysConfig[fieldName] = parseCommaSeparatedString(fieldValue);
```

Comment:
Not really sure why this was needed? All of them do the exact same thing so previous logic seemed to be fine.

Reply:
fixed, thanks